examples/connector/golang/golang_connector/main.go

```diff
@@ -30,7 +30,8 @@ func (s *server) Update(in *pb.UpdateRequest, stream pb.Connector_UpdateServer)
 	state := MyState{}
 	json.Unmarshal([]byte(state_json), &state)
 
-	log.Println("config: ", config, "selection: ", selection, "state_json: ", state_json, "mystate: ", state)
+	message := fmt.Sprintf("config: %s, selection: %s, state_json: %s, mystate: %s", config, selection, state_json, state)
```

Comment:
Introduce variables for INFO, WARNING and SEVERE similar to java?

Reply:
Added variables.